lib/fluent/plugin_helper/server.rb

```diff
@@ -370,6 +370,8 @@ def server_create_udp_socket(shared, bind, port)
         sock
       end
 
+      PEERADDR_FAILED = ["?", "?", "name resolusion failed", "?"]
```

Comment:
Very minor comment: remote_port usually returns Integer, but it returns String when failed. So it seems better to have comment about that it may return string at remote_port definition.